src/transformers/models/musicgen/modeling_musicgen.py

```diff
@@ -818,7 +1071,13 @@ def forward(
         if not return_dict:
             return tuple(
                 v
-                for v in [hidden_states, next_cache, all_hidden_states, all_self_attns, all_cross_attentions]
+                for v in [
+                    hidden_states,
+                    next_cache,
+                    all_hidden_states,
+                    all_self_attns,
+                    all_cross_attentions,
+                ]
```

Comment:
Should be a one liner
```suggestion
                for v in [hidden_states, next_cache, all_hidden_states, all_self_attns, all_cross_attentions]
```